redpanda/examples/python/kafka-produce.py

```diff
@@ -1,4 +1,13 @@
 #
+# To run this script, you need confluent-kafka libraries installed.
+# To create a dedicated venv for this, you can do:
+#
+# $ mkdir confluent-kafka; cd confluent-kafka
+# $ python3 -m venv confluent-kafka
+# $ cd confluent-kafka
+# $ source bin/activate
+# $ pip3 install confluent-kafka
+#
```

Comment:
Note, that on a Mac you may need to install the librdkafka package.  You can use "brew install librdkafka" if the pip3 command fails with an error like "librdkafka/rdkafka.h' file not found" as found at https://github.com/confluentinc/confluent-kafka-python/issues/166.